src/sounds.cpp

```diff
@@ -36,14 +66,14 @@ static std::vector<std::pair<tripoint, int>> recent_sounds;
 static std::vector<std::pair<tripoint, sound_event>> sounds_since_last_turn;
 // The sound events currently displayed to the player.
 static std::unordered_map<tripoint, sound_event> sound_markers;
+std::vector<sound_effect> sound_effects_p;
 
-void sounds::ambient_sound( const tripoint &p, int vol, std::string description )
-{
+void sounds::ambient_sound( const tripoint &p, int vol, std::string description ) {
```

Comment:
The official code style (doc/CODE_STYLE.txt) is to have the opening brace of the function definition on the next line.